build.sbt

```diff
@@ -130,7 +130,7 @@ lazy val docs = project
   .settings(common)
   .settings(dontPublish)
   .settings(
-    name := "Akka Persistence R2DBC",
+    name := "Akka Persistence plugin for R2DBC",
```

Comment:
for the record, I think the reason we use a readable name here is that it ends up as title or something in docs